tests/rules/no-duplicate-string.test.ts

```diff
@@ -20,6 +20,7 @@
 import { RuleTester } from "eslint";
 
 const ruleTester = new RuleTester({
+  parser: require.resolve("@typescript-eslint/parser"),
```

Comment:
can we use `RuleTesterJsWithTypes ` from Andrea?

Reply:
Well, the class is defined in SonarJS, therefore not available in eslint-plugin-sonarjs. However, we could temporarily duplicate its implementation here in eslint-plugin-sonarjs, create a ticket to remove the one in SonarJS, and eventually use eslint-plugin-sonarjs's one once we release it.

What do you think ?

Reply:
@saberduck ?

Reply:
yep, that could be a good idea, however maybe in separate PR, I would merge this as is